plugins/obs-ffmpeg/obs-nvenc-helpers.c

```diff
@@ -7,6 +7,8 @@
 
 #ifdef _WIN32
 #include <util/windows/device-enum.h>
+#else
+#include <GL/gl.h>
```

Comment:
This header is included without any check as a dependency in obs-ffmpeg CMake.

Reply:
So it is, this was just taken from the original implementation, I thinkg the correct thing to do here would be to link against `OBS::glad` and use `#include <glad/glad.h>` instead?

All that's really used from the header is `GL_TEXTURE_2D` and `GLuint` anyway.

Reply:
Depending only on a header, is still a dependency.

Reply:
I think it's `find_package(OpenGL REQUIRED)` and `OpenGL::GL` for this case.